CMAKE static build issue

<!--
If you have a question rather than reporting a bug please go to
- https://gitter.im/OGRECave/ogre or
- https://forums.ogre3d.org/
where you get much faster responses.

This is a template helping you to create an issue which can be processed as quickly as possible.
-->
#### System Information
- Ogre Version: 14.3.1
- Operating System / Platform: Windows 10 x64
- RenderSystem: any
- cmake version 3.31.0-rc2

#### Detailled description
<!-- your description -->
When i try to build ogre into a static library using the following options and commands my build fails
```
set IRON_PROJECT_ROOT_PATH=%~dp0..\..\
CMAKE_BUILD_TYPE=Debug

rmdir /s /q %IRON_PROJECT_ROOT_PATH%built\assets\libraries\ogre-14.3.1
mkdir %IRON_PROJECT_ROOT_PATH%built\assets\libraries\ogre-14.3.1
mkdir %IRON_PROJECT_ROOT_PATH%built\assets\libraries\ogre-14.3.1\cmake
mkdir %IRON_PROJECT_ROOT_PATH%built\assets\libraries\ogre-14.3.1\result
cd %IRON_PROJECT_ROOT_PATH%libraries\ogre-14.3.1
cmake -S . -B %IRON_PROJECT_ROOT_PATH%built\assets\libraries\ogre-14.3.1\cmake -A x64 -G "Visual Studio 17 2022" -D OGRE_STATIC=TRUE -D BUILD_SHARED_LIBS=OFF -D CMAKE_BUILD_TYPE=Debug -D OGRE_CONFIG_DOUBLE=TRUE -D OGRE_BUILD_SAMPLES=FALSE -D CMAKE_INSTALL_PREFIX=%IRON_PROJECT_ROOT_PATH%built\assets\libraries\ogre-14.3.1\result
cmake --build %IRON_PROJECT_ROOT_PATH%built\assets\libraries\ogre-14.3.1\cmake --config Debug -j -v --target install
```

It throws an error that it can't find the file "C:/Users/8IRON/Desktop/game_cpp/built/assets/libraries/ogre-14.3.1/cmake/lib/OgreMainStatic_d.pdb", for some reason the file is in "C:/Users/8IRON/Desktop/game_cpp/built/assets/libraries/ogre-14.3.1/cmake/lib/Debug/OgreMainStatic_d.pdb" along with all the rest of the files
![Untitled](https://github.com/user-attachments/assets/7e68f4c0-39f9-4c39-b0a0-43b656725069)



#### Ogre.log
The issue seems to be the error below, but i attached the full cmake log [aaaa.txt](https://github.com/user-attachments/files/17788265/aaaa.txt)
```CMake Error at OgreMain/cmake_install.cmake:57 (file):
           file INSTALL cannot find
           "C:/Users/8IRON/Desktop/game_cpp/built/assets/libraries/ogre-14.3.1/cmake/lib/OgreMainStatic_d.pdb":
           File exists.
         Call Stack (most recent call first):
           cmake_install.cmake:145 (include)
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -368,7 +368,7 @@ cmake_dependent_option(OGRE_CONFIG_FILESYSTEM_UNICODE "paths expected to be in U
 cmake_dependent_option(OGRE_INSTALL_SAMPLES "Install Ogre demos." TRUE "OGRE_BUILD_SAMPLES" FALSE)
 option(OGRE_INSTALL_TOOLS "Install Ogre tools." TRUE)
 option(OGRE_INSTALL_DOCS "Install documentation." TRUE)
-cmake_dependent_option(OGRE_INSTALL_PDB "Install debug pdb files" TRUE "MSVC" FALSE)
+cmake_dependent_option(OGRE_INSTALL_PDB "Install debug pdb files" TRUE "MSVC;NOT OGRE_STATIC" FALSE)
 option(OGRE_PROFILING "Enable internal instrumentation." FALSE)
 set(OGRE_PROFILING_REMOTERY_PATH "" CACHE PATH "set this to Remotery/lib to use Remotery instead of the buildin profiler")
 cmake_dependent_option(OGRE_CONFIG_STATIC_LINK_CRT "Statically link the MS CRT dlls (msvcrt)" FALSE "MSVC" FALSE)

